Python/Product/Debugger/Debugger.csproj

```diff
@@ -239,6 +245,7 @@
       <SubType>Designer</SubType>
     </None>
     <Content Include="DkmDebugger\PythonDkm.natvis" />
+    <None Include="packages.config" />
```

Comment:
Don't add nuget dependencies directly to the project. Add them to the `Build\15.0\packages.config` file and run `PreBuild.ps1` again.